src/table/sticky-scrollbar.tsx

```diff
@@ -19,18 +17,15 @@ export default forwardRef(StickyScrollbar);
 function StickyScrollbar({ wrapperRef, tableRef, onScroll }: StickyScrollbarProps, ref: React.Ref<HTMLDivElement>) {
   const scrollbarRef = React.useRef<HTMLDivElement>(null);
   const scrollbarContentRef = React.useRef<HTMLDivElement>(null);
-  const isRefresh = useVisualRefresh();
   const mergedRef = useMergeRefs(ref, scrollbarRef);
 
   /**
    * Use the appropriate AppLayout context (Classic or Visual Refresh) to determine
    * the offsetBottom value to be used in the useStickyScrollbar hook.
    */
-  const { stickyOffsetBottom: offsetBottomClassic } = useAppLayoutContext();
-  const { offsetBottom: offsetBottomVisualRefresh } = useContext(AppLayoutContext);
-  const offsetBottom = isRefresh ? offsetBottomVisualRefresh : offsetBottomClassic;
+  const { stickyOffsetBottom } = useAppLayoutContext();
```

Comment:
Good simplification here. Instead of reading two contexts and reconciling the value, we expect the correct value to be provided by the respective context